safe-global/web-core
# feat: Safe Apps permissions

## What it solves

Resolves https://github.com/safe-global/web-core/issues/589

## How this PR fixes it
This PR includes the permission system for Safe Apps:

- Adds permissions management support for the safe apps sdk [EIP-2255](https://eips.ethereum.org/EIPS/eip-2255) implementation released on [7.6.0](https://www.npmjs.com/package/@gnosis.pm/safe-apps-sdk/v/7.6.0)
- Adds permission support for allowing [Browser features](https://www.permissionspolicy.com/) in Safe Apps 

[Designs](https://www.figma.com/file/RqEZSC19sJRH6rbOFm6kxj/Assistance-pop-ups?node-id=222%3A2298)
[RFC](https://docs.google.com/document/d/1hHjoHnwjhq3xtyBuIogFkglS9f6GLsbZTbqZsxduVNE)

## How to test it
Check the tests included here https://github.com/safe-global/safe-react/pull/4019
